tests/detect-engine-proto/test.rules

```diff
@@ -0,0 +1,3 @@
+alert udp any any -> any any (msg:"Not tcp"; flow:to_server; sid:1;)
```

Comment:
I think that it's important to add a filter to ensure we get zero matches for this rule, otherwise it seems lost in this test...

Reply:
Agreed. Let's try to target all `sid`s that we use.

Reply:
Ok. Should I go-ahead to add the rule for HTTP?

Reply:
I'm not sure I understand the need for http rule, could you please explain?
Here in the above comments, we are suggesting to add a filter for sid:1 making sure alert count stays 0.

Reply:
I think the comment is because of this "Got wondering if we should add a rule to match on http as well, since we have that traffic in the pcap?", @inashivb. :P

Reply:
oh! missed that. Sure. :)